src/main/java/crazypants/enderio/machine/slicensplice/ContainerSliceAndSplice.java

```diff
@@ -51,9 +52,9 @@ public void putStack(@Nullable ItemStack stack) {
   public static int NUM_INVENTORY_SLOT = 4 * 9;
 
   static private final Item[] slotItems1 = { Items.WOODEN_AXE, Items.STONE_AXE, Items.IRON_AXE, Items.GOLDEN_AXE,
```

Comment:
better change it to a Things while we're at it. Also make sure the Background slot it goes into gets the whole list and not one random one. (new functionality that wasn't propagated to all GUIs)